node/mock-test.ts

```diff
@@ -211,7 +211,7 @@ export class MockTestRunner {
 
     // Downloads the specified node version to the download destination. Returns a path to node.exe
     private downloadNode(nodeVersion: string, downloadDestination: string): string {
-        shelljs.rm('-rf', downloadDestination);
+        task.rmRF(downloadDestination);
```

Comment:
Do we want to have these hard coded to functions (rmRf, mkdirP) or keep the syntax similar to what it was?

Reply:
I'm not sure I understand what you're asking here - are you concerned about us using functions from task here? Or is it something else?

Reply:
Ignore this comment, I misunderstood the surface area we expose.